wpeverest/everest-forms
# Fix - Prevent more than one Keyboard Shortcut keys pop-up modal opening at a time.

### All Submissions:






### Changes proposed in this Pull Request:



 Opening of more than one Keyboard Shortcut keys pop-up modal is prevented at a time.

### How to test the changes in this Pull Request:

1. Go to Everest Forms builder.
2. Verify, that there is no more than one Keyboard Shortcut keys pop-up modal when using a keyboard shortcut key Ctrl + h multiple times.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Prevent more than one Keyboard Shortcut keys pop-up modal opening at a time.
